scripts-setup/create-distro-layer

```diff
@@ -0,0 +1,97 @@
+#! /bin/bash
+
+set -e
+
+PROGNAME=$(basename $0)
+
+usage()
+{
+    cat >&2 <<EOF
+Usage: $PROGNAME --distro <DISTRO> [<options>]
+Options:
+    -h, --help         Print this usage message
+    -d, --distro       Set the DISTRO name for newly created distro layer
+    -m, --mender       Base the layer on mender configurations
+    -c, --color        Colorize the output; can be 'never', 'always',
+                             or 'auto' (default 'auto')
+Arguments:
+Examples:
+- To create a new distro layer without mender configurations:
+  $ $PROGNAME --distro testdistro
+EOF
+}
+
+DISTRO_NAME=""
+MENDER=false
+
+# get command line options
+SHORTOPTS="hmd:"
+LONGOPTS="help,mender,distro:"
+
+ARGS=$(getopt --options $SHORTOPTS --longoptions $LONGOPTS --name $PROGNAME -- "$@" )
+if [ $? != 0 ]; then
+   usage
+   exit 1
+fi
+
+eval set -- "$ARGS"
+while true;
+do
+    case $1 in
+        -h | --help)       usage; exit 0 ;;
+        -m | --mender)     MENDER=true; shift;;
+        -d | --distro)     DISTRO_NAME="$2"; shift 2;;
+        -- )               shift; break ;;
+        * )                break ;;
+    esac
+done
+
+if [ -z "$DISTRO_NAME" ]; then
+    usage
+    echo "ERROR: You must specify DISTRO." >&2
+    exit 1
+fi
+
+TOP_DIR=$(dirname $(dirname $0))
+cd ${TOP_DIR}
+LAYER_DIR="layers/meta-${DISTRO_NAME}"
+
+if [[ -d layers/meta-tegrademo ]]; then
```

Comment:
The `[[ ]]` construct is a bashism.  It would be better to stick with POSIX standard shell syntax here and in the rest of the script.  The standard single brackets `[ ]` should work just as well.